webapp/src/store/workspace.ts

```diff
@@ -15,9 +17,14 @@ export const fetchWorkspace = createAsyncThunk(
     async () => client.getWorkspace(),
 )
 
+type WorkspaceState = {
+    value: IWorkspace|null
```

Comment:
maybe this should be now `current` instead of `value`.

Reply:
Done.